readme.md

```diff
@@ -18,7 +18,7 @@ npm install
 
 ## Adding credentials
 
-You'll need access to the [Design System team credentials](https://github.com/alphagov/design-system-team-credentials) repository.
+You'll need access to BitWarden to access the credentials for ZenDesk.
```

Comment:
The DS cred repo had further instructions, but this feels more like a dead end. Do we need to add a "contact a team member for access" bit to provide the next action?

Reply:
I've updated the PR to include a bit more detail in the `.env.example` – reckon that's sufficient?